File: Java/Car.java
package Java;

public class Car {
    Integer id;
    String licence;
    Account driver;
    Integer passenger;

    public Car(String license, Account driver) {
        this.licence = license;
        this.driver = driver;
    }

    void printDataCar() {
        System.out.println("License: " + licence + " Driver: " + driver.name);
    }
}

File: JS/Car.js
function Car(license, driver) {
    this.id;
    this.licence = license;
    this.driver = driver;
    this.passenger;
}

Car.prototype.printDatacar = function () {
    console.log(this.driver);
    console.log(this.driver.name);
    console.log(this.driver.document);
}